packages/web-config/webpack.config.base.ts

```diff
@@ -74,7 +74,7 @@ const getCommonConfig = () => {
                 [require.resolve('@babel/plugin-proposal-class-properties'), { loose: true }],
               ],
               presets: [
-                require.resolve('@babel/preset-env'),
+                [require.resolve('@babel/preset-env'), { targets: { 'chrome': 63 } }],
```

Comment:
@flotwig

Reply:
so... this will transpile to code targeting Chrome >= 63?

what about for firefox?

Reply:
well they both should have similar feature support, but I could add firefox in there as well, maybe `firefox: 60` ?